lib/OpenQA/UserAgent.pm

```diff
@@ -38,6 +40,7 @@ sub new {
         for my $path (@cfgpaths) {
             my $file = $path . '/client.conf';
             next unless $file && -r $file;
+            log_info("Reading config from $file");
```

Comment:
not sure if this is a good idea to print output by default when AFAIK the client does not offer an interface to increase/decrease the log level. I think "log_debug" + command line parameters to enable debug messages would be a better approach

Reply:
Yeah, this actually resulted in https://progress.opensuse.org/issues/88053
So we have to find an alternative

Reply:
What about just writing it to stderr? Users would usually only pass stdout to their JSON parsers.